src/sdk/model/o365/o365-organization.ts

```diff
@@ -0,0 +1,221 @@
+import { Entity } from '../common/entity';
+import { EntityType } from '../common/__json__/entity-type';
+import { O365OrganizationJson, O365OrganizationRegion, O365OrganizationType } from './__json__/o365-organization-json';
+import { Iland } from '../../iland';
+import { O365JobJson } from './__json__/o365-job-json';
+import { O365Job } from './o365-job';
+import { O365RestoreSession } from './o365-restore-session';
+import { O365RestoreSessionJson } from './__json__/o365-restore-session-json';
+import { O365RestoreSessionStartRequest } from './o365-restore-session-start-request';
+import { O365User } from './o365-user';
+import { O365UserJson } from './__json__/o365-user-json';
+
+/**
+ * O365 Organization
+ */
+export class O365Organization extends Entity {
+
+  constructor(private _json: O365OrganizationJson) {
+    super(_json);
+  }
+
+  /**
+   * Gets a O365 Organization by UUID
+   * @param uuid
+   * @returns {Promise<O365Organization>} promise that resolves with a O365 Organization
+   */
+  static async getO365Organization(uuid: string): Promise<O365Organization> {
+    return Iland.getHttp().get(`/o365-organizations/${uuid}`).then((response) => {
```

Comment:
half of the `promise.then()` calls in this PR use `response` and half use `res` to point to the resolved API response so i would suggest picking one variable name just for style consistency. i personally like the full word `response` bc it's very descriptive and short enough